cypress/e2e/hub/repositories.cy.ts

```diff
@@ -23,6 +23,28 @@ describe('Repositories', () => {
     cy.get('[data-cy="alert-toaster"]').should('be.visible');
     cy.galaxykit('-i repository delete ' + repositoryName);
   });
+  it('should sync repository', () => {
+    const remoteName =
+      'repositories_repository_' + randomString(6, undefined, { isLowercase: true });
+    cy.createRemote(remoteName);
```

Comment:
@ZitaNemeckova we could do something as you did in other tests...

```suggestion
    cy.createRemote(remoteName).then((remote) => {
    
  ... rest of the test
  
  ```
    
    ;
```